### DISCUSSION TITLE
[Feature] Reverse default left/right pos. when merging people (UX only)

### DISCUSSION THREAD

**Discussion by LaurentAtJoes** [immich-app/immich#7241] [Feature Request]
### The feature

Maybe there's use cases that I don't see here, but from my own experience, but the most common situation of merging people is the following :
you're on a face for [untagged person], and want to merge it with an existing tagged person.

Thing is, when selecting "merge..." from the menu, the default action proposed is to merge the existing person into the untagged one.

I reckon there are situations when you start from _the existing person_, but imho most of the time you're in the above one.
So I think the "default" workflow should be "merge _the currently displayed_ person _into an existing_  one.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by YuriyGural** [comment]
_I typed the text and decided to look for a similar topic._ :-)

Face recognition in Immich is amazing, for example in comparison with recognition in DS photo.

When you merge people together, what is your sequence of actions?
You already have a group of people to whom you have given a name. You scroll through the list of unrecognized faces and find a duplicate, then open the photos to check and decide to merge that face with an existing face. After selecting an existing face, the screen suggests combining the existing face with a new one, which is not logical. After this, you need to click the swap button each time.

You are absolutely right, need to put a new face on the left to be absorbed into the existing one, which on the right.
